skimmer/skimmer_utils.py

```diff
@@ -93,3 +94,7 @@ def make_pt_eta_phi_mass_lorentz_vector(pt, eta=None, phi=None, mass=None):
 def __get_number_of_events(events):
     return ak.sum(events.Weight)
 
+def get_cutFlow_from_skims(input_file, cut_flow_tree):
```

Comment:
Can you name the function `get_cut_flow_from_skims` to follow the naming convention? :)